plugins/home/src/homePageComponents/WelcomeTitle/timeUtil.ts

```diff
@@ -32,12 +35,26 @@ export function getTimeBasedGreeting(): { language: string; greeting: string } {
       greeting: 'Get some rest',
     };
   }
+
   const timeOfDay = (hour: number): { [language: string]: string } => {
     if (hour < 12) return goodMorning;
     if (hour < 17) return goodAfternoon;
     return goodEvening;
   };
+
   const greetings = timeOfDay(currentHour);
+
+  if (Array.isArray(language) && language.length > 0) {
+    const validLanguages = language.filter(lang => lang && greetings[lang]);
```

Comment:
Need help here. How can I introduce log here, when I do "console.log" it prints the error in browser. But I guess the expectation is to have it with application logs.
@awanlin @camilaibs

Reply:
I was thinking a simple console is enough, in case developers inspect the page to see what's going on, but a totally optional suggestion, we can merge without it 🙂